src/app/task-displayer/task-displayer.component.ts

```diff
@@ -16,21 +15,11 @@ import { TaskData } from '../taskdata';
 })
 export class TaskDisplayerComponent {
 
-  taskList$: Observable<TaskData[]> = this.taskService.taskList$;
-
-
+  taskList$: Observable<TaskList>;
 
   constructor(private taskService: TaskService) {
-    this.taskService.taskList$.subscribe({
-      next: (tasks: TaskData[]) => {
-      },
-      error: (error) => {
-        console.error(error);
-      }
-    });
-
-    this.taskService.loadTasksFromAPI();
-  };
+    this.taskList$ = this.taskService.loadTasksFromAPI();
```

Comment:
this works, but keep in mind that in the service you have taskList$ which you don't use